src/operator/bufferTime.ts

```diff
@@ -120,7 +120,7 @@ class BufferTimeSubscriber<T> extends Subscriber<T> {
               private scheduler: Scheduler) {
     super(destination);
     const context = this.openContext();
-    this.timespanOnly = bufferCreationInterval == null || bufferCreationInterval < 0;
+    this.timespanOnly = bufferCreationInterval === null || bufferCreationInterval < 0;
```

Comment:
Help me understand plz, why is there a notable difference between `undefined` and `null` here? i.e. why the fact that `== null` would match `undefined` too is a problem?